src/test/java/be/syntra/devshop/DevshopFront/services/SearchServiceTest.java

```diff
@@ -3,35 +3,43 @@
 import be.syntra.devshop.DevshopFront.models.SearchModel;
 import be.syntra.devshop.DevshopFront.testutils.JsonUtils;
 import be.syntra.devshop.DevshopFront.testutils.TestWebConfig;
+import be.syntra.devshop.DevshopFront.testutils.WebContextTestExecutionListener;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.web.client.RestClientTest;
 import org.springframework.context.annotation.Import;
+import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.support.DependencyInjectionTestExecutionListener;
+import org.springframework.test.context.support.DirtiesContextTestExecutionListener;
 import org.springframework.test.web.client.MockRestServiceServer;
 import org.springframework.web.client.RestTemplate;
 
 import java.math.BigDecimal;
 
+import static junit.framework.TestCase.*;
 import static org.assertj.core.api.Assertions.assertThat;
 
 @RestClientTest(SearchServiceImpl.class)
 @ExtendWith(MockitoExtension.class)
 @Import({TestWebConfig.class, JsonUtils.class})
-public class SearchServiceTest {
+@TestExecutionListeners({WebContextTestExecutionListener.class,
+        DependencyInjectionTestExecutionListener.class,
+        DirtiesContextTestExecutionListener.class})
+class SearchServiceTest {
 
     @Autowired
     private RestTemplate restTemplate;
 
     @Autowired
     private SearchServiceImpl searchService;
 
-    MockRestServiceServer mockServer;
+    private MockRestServiceServer mockServer;
```

Comment:
you no longer need the RestTemple, MockRestServiceServer, or the BeforeEach in order for those tests to succeed :)

Reply:
removed + removed extendsWith annotation on testClass